Source/WebCore/page/CaptionUserPreferences.h

```diff
@@ -44,18 +44,20 @@ class AudioTrackList;
 class TextTrackList;
 struct MediaSelectionOption;
 
+enum class CaptionUserPreferencesCaptionDisplayMode : uint8_t {
```

Comment:
I think we should call this `CaptionUserPreferencesDisplayMode`. No need to repeat `Caption`.

Reply:
OK.